ocis-pkg/natsjsregistry/registry.go

```diff
@@ -180,6 +186,10 @@ func (n *storeregistry) storeOptions(opts registry.Options) []store.Option {
 		natsjskv.EncodeKeys(),
 	}
 
+	if defaultTTL, ok := opts.Context.Value(defaultTTLKey{}).(time.Duration); ok {
+		storeoptions = append(storeoptions, natsjskv.DefaultTTL(defaultTTL))
+	}
```

Comment:
As mentioned above, we are following the naming from the nats Store implementation